vkdgen.py

```diff
@@ -113,7 +113,9 @@ def convertTypeArray(typ, name):
 class DGenerator(OutputGenerator):
 	def __init__(self, errFile=sys.stderr, warnFile=sys.stderr, diagFile=sys.stderr):
 		super().__init__(errFile, warnFile, diagFile)
-	
+		self.enumConstants = []
+		self.funcNames = []
+

```

Comment:
The bindings are not intended to be readable. You can look at the C headers for a cleaner reference.